Apps/Common/src/Swagger/ConfigureSwaggerGenOptions.cs

```diff
@@ -67,6 +68,15 @@ public void Configure(SwaggerGenOptions options)
                 Scheme = "bearer",
             });
 
+            options.AddSecurityDefinition("apikey", new OpenApiSecurityScheme
+            {
+                Name = ApiKeyRequirement.ApiKeyHeaderNameDefault,
+                Description = $"Authorization using the {ApiKeyRequirement.ApiKeyHeaderNameDefault} header. Example: \"{ApiKeyRequirement.ApiKeyHeaderNameDefault} {{apiKey}}\"",
```

Comment:
Where is `{{apiKey}}` coming from?

Reply:
Scratch that. I realized that it's not outputting a variable named apiKey, but outputting the literal text {apiKey} by escaping the brackets.